db-client-java/src/main/java/com/eventstore/dbclient/HttpUtils.java

```diff
@@ -0,0 +1,140 @@
+        PersistentSubscriptionInfo info = new PersistentSubscriptionInfo();
+
+        info.setEventStreamId(node.get("eventStreamId").asText());
+        info.setGroupName(node.get("groupName").asText());
+        info.setStatus(node.get("status").asText());
+        info.setAverageItemsPerSecond(node.get("averageItemsPerSecond").asDouble());
+        info.setTotalItemsProcessed(node.get("totalItemsProcessed").asLong());
+        info.setLastKnownEventNumber(node.get("lastKnownEventNumber").asLong());
+
+        if (node.get("connectionCount") != null) {
+            info.setConnectionCount(node.get("connectionCount").asLong());
+        } else {
+            info.setConnectionCount(0);
+        }
+
+        info.setTotalInFlightMessages(node.get("totalInFlightMessages").asLong());
+        info.setConfig(parseConfig(node.get("config")));
+
+        return info;
+    }
+
+    public static PersistentSubscriptionConfig parseConfig(JsonNode node) {
+        PersistentSubscriptionConfig config = null;
+
+        if (node != null) {
+            config = new PersistentSubscriptionConfig();
+            config.setResolveLinktos(node.get("resolveLinktos").asBoolean());
+            config.setStartFrom(node.get("startFrom").asLong());
+            config.setMessageTimeoutMilliseconds(node.get("messageTimeoutMilliseconds").asLong());
+            config.setExtraStatistics(node.get("extraStatistics").asBoolean());
+            config.setMaxRetryCount(node.get("maxRetryCount").asLong());
+            config.setLiveBufferSize(node.get("liveBufferSize").asLong());
+            config.setBufferSize(node.get("bufferSize").asLong());
+            config.setReadBatchSize(node.get("readBatchSize").asLong());
+            config.setPreferRoundRobin(node.get("preferRoundRobin").asBoolean());
+            config.setCheckPointAfterMilliseconds(node.get("checkPointAfterMilliseconds").asLong());
+            config.setMinCheckPointCount(node.get("minCheckPointCount").asLong());
+            config.setMaxCheckPointCount(node.get("maxCheckPointCount").asLong());
+            config.setMaxSubscriberCount(node.get("maxSubscriberCount").asLong());
+            config.setNamedConsumerStrategy(ConsumerStrategy.valueOf(node.get("namedConsumerStrategy").asText()));
+        }
+
+        return config;
+    }
+
+    public static PersistentSubscriptionInfo parseInfoFromWire(Persistent.SubscriptionInfo wire) {
+        PersistentSubscriptionInfo info = new PersistentSubscriptionInfo();
+
+        info.setEventStreamId(wire.getEventSource());
+        info.setGroupName(wire.getGroupName());
+        info.setConnectionCount(wire.getConnectionsCount());
+        info.setStatus(wire.getStatus());
+        info.setAverageItemsPerSecond(wire.getAveragePerSecond());
+
+        try {
+            info.setLastKnownEventNumber(Integer.parseInt(wire.getLastKnownEventPosition()));
+        } catch (NumberFormatException e) {
+            // TODO - Support log position parsing.
```

Comment:
Will this be done in the next PR handling the `PersistentSubscriptionInfo` name changes?

Reply:
Yes